devices.json

```diff
@@ -1,6 +1,65 @@
 {
   "version": 0.1,
   "devices": [
+    {
+      "vendor": "TROLINK",
+      "bDetailed": "0",
+      "name": "Smart IOT control board",
+      "model": "SM-028_V1.3",
+      "chip": "BL602",
+      "board": "SM-028_V1.3",
+      "pins": {
+        "1":  "Button;1",
+        "2":  "Button;2",
+        "3":  "Button;3",
+        "4":  "Button;4",
+        "14": "Button;5",
+        "17": "Button;6",
+        "20": "Button;7",
+        "21": "Button;8",
+        "22": "Button;9"
+      },
+      "keywords": ["", ""],
+      "image": "https://user-images.githubusercontent.com/14217781/209668729-d62d290f-33c7-4bda-9696-7a16734e841a.jpg",
+      "wiki": "",
+      "product": "https://world.taobao.com/item/691740234866.htm"
+    },
+    {
+      "vendor": "QIACHIP",
+      "bDetailed": "0",
+      "name": "WiFi DIY Relay",
+      "model": "KR05-1CHKG-W",
+      "chip": "BL602",
+      "board": "SM-028_V1.3",
+      "pins": {
+        "20": "Relay;1",
+        "4":  "Button;1",
```

Comment:
"Button;1" Should be "Pin" GPIO_14, the actual chip pin 4.

`"14":  "Button;1",`

It's confusing that are named "pin" instead of GPIO.
Same for the Configure Module page where are labeled "Px" without more details.
Sorry.